docs/elasticsearch-specification.asciidoc

```diff
@@ -517,7 +518,7 @@ By default, the readiness probe checks that the pod can successfully respond to
 [source,yaml]
```

Comment:
I think this needs to be `[source,yaml,subs="attributes"]` to sub the attr, see here:
http://cloud-on-k8s_2274.docs-preview.app.elstc.co/guide/en/cloud-on-k8s/master/k8s-prestop.html

Reply:
Nice, thanks!